aliss/templates/homepage.html

```diff
@@ -66,3 +66,58 @@ <h3>Join the ALISS community</h3>
   {% include 'partials/share-search.html' %}
 </main>
 {% endblock %}
+{% block before_body_close %}
+  <script type="text/javascript">
+    $(document).ready(function(){
+      var options = {
+
+        url: function(phrase) {
```

Comment:
This option seems to define a GET request url when you type in 3 characters. I think this `url` is supposed to be for getting a list of potential options, which right now we don't need as they are defined in `data` .

Reply:
Is it safe to remove?

Reply:
I meant to ask you about this section. When we were discussing this you mentioned you wanted the phrase to be returned for analytics I think? There was an example you saw which had the correct sort of behaviour so I tried to copy that although I was meaning to ask for more information to see whether it was what you were looking for.

Reply:
ok doke, it was possibly in relation to using the API, and then tracking API use. For now, I'll be using a separate bit of js to do tracking so sounds like its safe to remove.

Reply:
Sounds good